src/Sylius/Bundle/ApiBundle/PropertyInfo/Extractor/EmptyPropertyListExtractor.php

```diff
@@ -15,6 +15,14 @@
 
 use Symfony\Component\PropertyInfo\PropertyListExtractorInterface;
 
+trigger_deprecation(
+    'sylius/api-bundle',
+    '1.14',
+    'The "%s" class is deprecated and will be removed in Sylius 2.0.',
+    EmptyPropertyListExtractor::class,
+);
+
+/** @deprecated since Sylius 1.14 and will be removed in Sylius 2.0. */
```

Comment:
This service has not yet been removed in 2.0, so it is already too late to deprecate and remove it